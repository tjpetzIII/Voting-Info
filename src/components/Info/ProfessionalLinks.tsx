import React from "react";

const ProfessionalLinks = () => {
  return (
    <nav className="flex items-center gap-2 text-xl mt-2">
      {[
        { label: "GitHub", href: "https://github.com/tjpetzIII" },
        { label: "LinkedIn", href: "https://www.linkedin.com/in/tpetz1/" },
      ].map((link, idx, arr) => (
        <React.Fragment key={link.href}>
          <a
            href={link.href}
            className="hover:text-blue-main hover:underline transition-colors duration-200"
          >
            {link.label}
          </a>
          {idx < arr.length - 1 && <span className="text-gray-400">|</span>}
        </React.Fragment>
      ))}
    </nav>
  );
};
export default ProfessionalLinks;
