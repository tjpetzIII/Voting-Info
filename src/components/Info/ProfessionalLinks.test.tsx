import React from "react";
import { render } from "@testing-library/react";
import ProfessionalLinks from "./ProfessionalLinks";

describe("ProfessionalLinks", () => {
  it("matches the snapshot", () => {
    const { asFragment } = render(<ProfessionalLinks />);
    expect(asFragment()).toMatchSnapshot();
  });
});
