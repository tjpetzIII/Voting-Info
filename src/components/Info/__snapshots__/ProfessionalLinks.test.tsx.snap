// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfessionalLinks matches the snapshot 1`] = `
<DocumentFragment>
  <nav
    class="flex items-center gap-2 text-xl mt-2"
  >
    <a
      class="hover:text-blue-main hover:underline transition-colors duration-200"
      href="https://github.com/tjpetzIII"
    >
      GitHub
    </a>
    <span
      class="text-gray-400"
    >
      |
    </span>
    <a
      class="hover:text-blue-main hover:underline transition-colors duration-200"
      href="https://www.linkedin.com/in/tpetz1/"
    >
      LinkedIn
    </a>
  </nav>
</DocumentFragment>
`;
